modules/tinymce/src/themes/silver/main/ts/modes/Iframe.ts

```diff
@@ -100,7 +100,7 @@ const render = (editor: Editor, uiComponents: RenderUiComponents, rawUiConfig: R
     OuterContainer.setSidebar(
       outerContainer,
       rawUiConfig.sidebar,
-      Options.getSidebarShow(editor)
+      Options.getSidebarShow(editor).fold(() => undefined, (value) => value.toLowerCase())
```

Comment:
Wondering if this would be clearer?
```suggestion
      Options.getSidebarShow(editor).map((value) => value.toLowerCase()).getOrUndefined()
```